docs/en/rst/installing/mysql.rst

```diff
@@ -5,8 +5,7 @@ MySQL
 
 It is strongly advised to use MariaDB which is a drop-in replacement for
```

Comment:
We should probably remove the "drop-in replacement" bit. As of 10.6 there's enough differences that it's really not (which is why it has its own driver now).